src/Microsoft.Extensions.Logging.AzureWebAppDiagnostics/Internal/WebAppLogConfiguration.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.AzureWebAppDiagnostics.Internal
+{
+    /// <summary>
+    /// Represents the configuration of the logger from Azure WebApp.
+    /// </summary>
+    public class WebAppLogConfiguration
+    {
+        /// <summary>
+        /// The configuration that has all loggers disabled.
+        /// </summary>
+        public static WebAppLogConfiguration Disabled { get; } = new WebAppLogConfiguration(isRunningInWebApp: false);
+
+        /// <summary>
+        /// Creates a new instance of the <see cref="WebAppLogConfiguration"/> class.
+        /// </summary>
+        /// <param name="fileLoggingEnabled">True if file logging is enabled, false otherwise</param>
+        /// <param name="fileLoggingLevel">The logging level for the file logger</param>
+        /// <param name="fileLoggingFolder">The folder where the file logger stores the logs</param>
+        /// <param name="blobLoggingEnabled">True if blob logging is enabled, false otherwise</param>
+        /// <param name="blobLoggingLevel">The logging level for the blob logger</param>
+        /// <param name="blobContainerUrl">The SAS endpoint where blob logs are stored</param>
+        public WebAppLogConfiguration(
```

Comment:
We don;t use default parameters in any public APIs because of versioning issues. Change this to a builder or just an options object with properties.